# Protein Contaminant Libraries for DDA and DIA Proteomics

Mass spectrometry-based proteomics is constantly challenged by the presence of contaminant background signals. In particular, protein contaminants from reagents and sample handling are almost impossible to avoid. For data-dependent acquisition (DDA) proteomics, an exclusion list can be used to reduce the influence of protein contaminants. However, protein contamination has not been evaluated and is rarely addressed in data-independent acquisition (DIA). How protein contaminants influence proteomic data is also unclear. In this study, we established new protein contaminant FASTA and spectral libraries that are applicable to all proteomic workflows and evaluated the impact of protein contaminants on both DDA and DIA proteomics. We demonstrated that including our contaminant libraries can reduce false discoveries and increase protein identifications, without influencing the quantification accuracy in various proteomic software platforms. With the pressing need to standardize proteomic workflow in the research community, we highly recommend including our contaminant FASTA and spectral libraries in all bottom-up proteomic data analysis. Our contaminant libraries and a step-by-step tutorial to incorporate these libraries in various DDA and DIA data analysis platforms can be valuable resources for proteomic researchers, freely accessible at https://github.com/HaoGroup-ProtContLib.

LC-MS raw data files for Contaminant-only samples can be found in ProteomeXchange Consortium with the data identifier, PXD031139 (http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD031139)

Please cite our publication: Frankenfield AM, Ni J, Ahmed M, Hao L. Protein contaminants matter: building universal protein contaminant libraries for DDA and DIA proteomics. Journal of proteome research. 2022 Jul 6;21(9):2104-13. 
https://pubs.acs.org/doi/full/10.1021/acs.jproteome.2c00145

---------------------------------------------------------------------------------------------------------------



About HaoGroup:


We are an academic research group focused on biological mass spectrometry at the Chemistry Department in George Washington University, Washington, D.C. 

Quantitative analysis of biomolecules offers a wealth of clues to understand cell biology and disease mechanisms. Mass spectrometry (MS) has become a central technology to study biomolecules such as proteins, peptides, lipids, and small molecule metabolites. Our research focuses on developing novel and improved bioanalytical methods using LC-MS platforms and applying the combination of analytical chemistry, bioinformatics, and cell biology approaches to study human diseases, in particular, neurodegenerative diseases. 

PI: Ling Hao (Ph.D.)

Hao Lab website: https://blogs.gwu.edu/haolab/


