# Protein Contaminant Libraries for DDA and DIA Proteomics

Mass spectrometry-based proteomics is constantly challenged by the presence of contaminant background signals. In particular, protein contaminants from reagents and sample handling are often abundant and almost impossible to avoid, compromising the sensitivity, specificity and reproducibility of protein identification and quantification. For data-dependent acquisition (DDA) proteomics, exclusion lists and contaminant FASTA libraries can be used to reduce the influences of protein contaminants. However, protein contamination has not been evaluated and is rarely addressed in data-independent acquisition (DIA) proteomics. In this study, we established protein contaminant FASTA and spectral libraries for bottom-up proteomics, and evaluated the impact of protein contaminants on both DDA and DIA proteomics. We found that including our contaminant library can reduce false identifications and increase protein IDs, without influencing the quantification accuracy. We evaluated our FASTA and spectral libraries in various DIA and DDA data analysis platforms and provided practical suggestions and step-by-step protocols on how to best incorporate the contaminant library for different software platforms. Therefore, we recommend using our contaminant libraries for both DDA and DIA proteomics workflows. With the increasing popularity of DIA proteomics, our contaminant FASTA and spectral libraries can be an especially useful resource for the proteomics community.

LC-MS raw data files for Contaminant-only samples can be found in ProteomeXchange Consortium with the data identifier, PXD031139 (http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD031139)


---------------------------------------------------------------------------------------------------------------



About HaoGroup:


We are an academic research group focused on biological mass spectrometry at the Chemistry Department in George Washington University, Washington, D.C. 

Quantitative analysis of biomolecules offers a wealth of clues to understand cell biology and disease mechanisms. Mass spectrometry (MS) has become a central technology to study biomolecules such as proteins, peptides, lipids, and small molecule metabolites. Our research focuses on developing novel and improved bioanalytical methods using LC-MS platforms and applying the combination of analytical chemistry, bioinformatics, and cell biology approaches to study human diseases, in particular, neurodegenerative diseases. 

PI: Ling Hao (Ph.D.)

Hao Lab website: https://blogs.gwu.edu/haolab/


